home-service/src/main/java/com/softserveinc/ita/homeproject/service/UserService.java

```diff
@@ -0,0 +1,20 @@
+package com.softserveinc.ita.homeproject.service;
+
+import com.softserveinc.ita.homeproject.model.CreateUser;
```

Comment:
The package isn't clear:
```
com.softserveinc.ita.homeproject.model.CreateUser
```

I think it should be `com.softserveinc.ita.homeproject.api.model.CreateUser`

Reply:
Why do we depend on api from service?

Reply:
It's the package, where we have all our DTOs etc.